.buildkite/pipeline.yml

```diff
@@ -66,9 +66,9 @@ steps:
     depends_on: js-bundles
     command: .buildkite/publish-react-native-ios-artifacts.sh
     artifact_paths:
-      - ios-xcframework/build/xcframeworks/*.zip
+      - ios-xcframework/build/xcframeworks/*.tar.gz
```

Comment:
Just a heads up that if we want to distribute this using SwiftPM we'll need to publish a ZIP version as well

Reply:
It should be trivial to add when the time comes. 

I haven't looked into that, but I assume we'll also need a different package structure (e.g. without `dummy.txt`). That, too, will be a matter of calling `zip` with the appropriate params.

Reply:
It seems a bit wasteful that same files are duplicated in two places, one in Buildkite's artifactory and the other in our public S3 bucket. It'd be nice if we can upload a "redirect link" as a buildkite artifact which takes use to the files in the public S3 bucket.

Reply:
Yep, not a blocker at all. I have had a quick research and don't think Buildkite supports uploading a "redirect link". But it does support [saving artifact to a S3 bucket](https://buildkite.com/docs/agent/v3/cli-artifact#using-your-private-aws-s3-bucket). Many we can configure the pipeline to tell Buildkite to save the tar.gz and podspec file to the correct location in our public artifacts S3 bucket.